README.rst

```diff
@@ -71,29 +76,41 @@ Installation
 
 Developer Environment
 ---------------------
+The project uses the modern `uv` package manager: https://docs.astral.sh/uv/
+There are a number of options that make life a lot easier in the Makefile.
+To see option for the Makefile run: `make`
 
 Follow these steps to set up your development environment to contribute to helpdesk:
  - check out the helpdesk app to your local file system::
 
     git clone https://github.com/django-helpdesk/django-helpdesk.git
  
- - install a virtual environment and activate it::
+ - Install the `uv` package manager::
+    https://docs.astral.sh/uv/getting-started/installation/
+
+ - install a virtual environment ::
   
-    python -m venv .venv && source .venv/bin/activate
+    uv venv
 
  - install the requirements for development::
 
-    pip install -r requirements.txt -r requirements-dev.txt
+    make develop
 
- - you can install the requirements for testing as well::
+ - you can install the requirements for using pinax teams using this commamnd::
 
-    pip install -r requirements-testing.txt
+    uv  sync  --all-extras --dev --group test --group teams
+    
+ - or teamss without the test files::
+    uv  sync  --all-extras --group teams
```

Comment:
This paragraph seems unclear. Maybe precise that you can install optional dependencies like `test` or `teams` by adding the `--group` argument and show these examples

`uv  sync  --all-extras --dev --group teams # Only teams` 
`uv  sync  --all-extras --dev --group test --group teams # For both`

Reply:
Good call - have changed